.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "CTF/crypto_lib"]
+	path = CTF/crypto_lib
+	url = https://github.com/sushshring/simplecrypto.git
```

Comment:
Should we make this repo public if we include it as a submodule?

Reply:
No haha, my thought was that the original C code that does the encryption should stay private and inaccessible unless you're working on developing the challenge itself :)